teams.json

```diff
@@ -38,6 +28,33 @@
         "GoogleCloudPlatform/stackdriver-sandbox"
       ]
     },
+    {
+      "name": "serverless",
+      "apis": [
+        "appengine",
+        "cloudfunctions",
+        "cloudscheduler",
+        "cloudtasks",
+        "container",
+        "run"
+      ]
+    },
+    {
+      "name": "devplat",
+      "apis": [
+        "artifactregistry",
+        "binaryauthorization",
+        "cloudbuild",
+        "cloudshell",
+        "container",
+        "containeranalysis",
+        "containerregistry"
+      ],
+      "repos": [
+        "GoogleCloudPlatform/bank-of-anthos",
```

Comment:
This is awesome.  Few things we should consider though - if we add these repositories here, they also need to be tracked in `repos.json`, setup a few bots (happy to do that), and make sure they're registered at devrel/repos.  Also - this will enable SLO tracking.  I would highly suggest doing an initial triage so the numbers don't go entirely out of whack.

Reply:
Okay, added to repos.json, and renamed 1 repo according to @simonz130